apps/public-docsite-v9/src/Concepts/Upgrade/FromV8/Components/RadioGroup.stories.mdx

```diff
@@ -0,0 +1,198 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Upgrading/from v8/Components/ChoiceGroup to RadioGroup Upgrade" />
+
+# ChoiceGroup to RadioGroup Upgrade
+
+Fluent UI v8 provides the `ChoiceGroup` control for presenting a list of radio options.
+In Fluent UI v9 `ChoiceGroup` is replaced with `RadioGroup`.
+
+While there are several differences between these controls, the primary change is that `RadioGroup` accepts
+its options as child `Radio` components while `ChoiceGroup` accepts options via its `options` prop.
+
+## Examples
+
+### Basic Upgrarde
+
+Basic usage of `ChoiceGroup` looks like
+
+```tsx
+import * as React from 'react';
+import { ChoiceGroup, IChoiceGroupOption } from '@fluentui/react/lib/ChoiceGroup';
+
+const ChoiceGroupBasicExample = () => {
+  const options: IChoiceGroupOption[] = [
+    { key: 'A', text: 'Option A' },
+    { key: 'B', text: 'Option B' },
+    { key: 'C', text: 'Option C', disabled: true },
+    { key: 'D', text: 'Option D' },
+  ];
+
+  return <ChoiceGroup defaultSelectedKey="B" options={options} label="Pick one" required={true} />;
+};
+```
+
+An equivalent `RadioGroup` usage is
+
+```tsx
+import * as React from 'react';
+import { Label, Radio, RadioGroup } from '@fluentui/react-components/unstable';
+import { useId } from '@fluentui/react-utilities';
+
+const RadioGroupBasicExample = () => {
+  const labelId = useId('label-');
+
+  return (
+    <>
+      <Label id={labelId} required>
+        Pick One
+      </Label>
+      <RadioGroup aria-labelledby={labelId} defaultValue="B">
+        <Radio value="A" label="A" />
```

Comment:
A small nitpick, but if the v8 option has `text: 'Option A'`, then the equivalent v9 `<Radio>` should have `label="Option A"` too

Reply:
Additionally, since the v8 example uses `required`, I think the v9 radio example should probably set it too.